samples/nfc/tnep_tag/src/main.c

```diff
@@ -149,15 +144,38 @@ static void nfc_callback(void *context, enum nfc_t4t_event event,
 	}
 }
 
+/** .. include_startingpoint_initial_msg_cb_rst */
+static int tnep_initial_msg_encode(struct nfc_ndef_msg_desc *msg)
+{
+	const uint8_t text[] = "Hello World";
+
+	NFC_NDEF_TEXT_RECORD_DESC_DEF(nfc_text, UTF_8, en_code,
+				      sizeof(en_code), text,
+				      sizeof(text));
```

Comment:
Would be better to use `strlen` to exclude the null terminator.

Reply:
Ok changed in all places